### PR TITLE
7506 accessibility datatypes 5 2

### PR DESCRIPTION
**Pull request by phudson-he**
### Types of changes

-   [ ] Bugfix (non-breaking change which fixes an issue)
-   [x] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Description of Change
Changes have been done to ensure [WCAG 2.1 requirements](https://www.w3.org/TR/WCAG21/#requirements-for-wcag-2-1) have been met for accessibility guidelines.

Details of the changes are as follows:

arches/app/templates/views/components/datatypes

- boolean.htm : Added aria-labels
- string.htm : Added aria-labels
- number.htm : Added aria-labels
- date.htm : Added aria-labels
- concept.htm : Added aria-labels and made keyboard operable
- domain-value.htm : Added aria-labels
- edtf.htm : Added aria-labels
- file-list.htm : Added aria-label
- geojson-feature-collection.htm : Added aria-labels and made keyboard operable
- node-value.htm : Added aria-labels
- Resource-instance.htm : Added aria-label and removed fixed width

### Issues Solved
Ensures that widgets (listed above) conform to accessibility guidelines
#

### Checklist

-   [x] Unit tests pass locally with my changes
-   [ ] I have added tests that prove my fix is effective or that my feature works
-   [ ] I have added necessary documentation (if appropriate)

#### Ticket Background
*   Sponsored by: Historic England
*   Found by: @phudson-he
*   Tested by: @phudson-he
*   Designed by: @phudson-he

### CODE REVIEW

**Comment by phudson-he** [comment]
@robgaston - This has now had dev/6.0.x merged in so now includes accessibility mode/css. Tests and checks all passed.

**Comment by csmith-he** [comment]
@robgaston @aarongundel  There don't appear to be any conflicts between base branch dev/6.1.x and the branch this PR relates to; are you happy to review?

**Review by aarongundel — CHANGES_REQUESTED** [review]
looks good!  Few minor changes....

File: arches/app/media/js/views/components/datatypes/concept.js [hunk 1]
```diff
@@ -12,6 +12,11 @@ define(['arches', 'knockout', 'viewmodels/concept-select'], function (arches, ko
                 this.multiple = ko.observable(false);
                 this.searchValue = ko.observable(filter.val || '');
                 this.node = params.node;
+                this.clearSelectionEventHandler = function(data,event){
```
Comment (csmith-he) [inline arches/app/media/js/views/components/datatypes/concept.js:15]: Hi @aarongundel - I've moved the select2Config.clear() from the template into the view model so it'll work for a click or a keyup when the enter key is pressed.

Are you happy with this pattern and the name of the function? If so, I'll apply the same pattern for all the other instances of this.

File: arches/app/templates/views/components/datatypes/node-value.htm [hunk 2]
```diff
@@ -46,10 +48,10 @@
 <!-- /ko -->
 
 <!-- ko if: $data.search -->
-<div class="col-md-4 col-lg-3">
+<div class="col-md-4 col-lg-3">aaa
```
[the remark this answers is not in the document]
Comment (csmith-he) [reply arches/app/templates/views/components/datatypes/node-value.htm]: Yes, they were left over from testing - they've now been removed

File: arches/app/media/js/views/components/datatypes/concept.js (hunk 1 above)
Comment (aarongundel) [reply arches/app/media/js/views/components/datatypes/concept.js:15]: @csmith-he - yep that'll work nicely!  Feel free to apply that to all the instances of this issue.

File: arches/app/templates/views/components/datatypes/resource-instance.htm [hunk 3]
```diff
@@ -1,8 +1,7 @@
 {% load i18n %}
 <!-- ko ifnot: $data.search -->
 <!--ko let: {self: $data} -->
-<div class="node-widget-label concept-label" style="width:540px;">
-
+<div class="node-widget-label concept-label" style="width:40vw;">
```
[the remark this answers is not in the document]
Comment (phudson-he) [reply arches/app/templates/views/components/datatypes/resource-instance.htm]: As per latest commit, changed back to previous value. But will revisit at a later date for graph designer responsiveness.

**Comment by phudson-he** [comment]
@aarongundel Forgive me for the reverts caused, commits were made by accident that were intended for another branch

**Comment by csmith-he** [comment]
@aarongundel - could you let me know which changes still need doing to allow this pull request to be approved and merged?

**Pull request closed without merging**